templates_ark/Clustering_cleanup.ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
<p>Should we add an example visualization for this one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Yeah agreed</p>